sched/task/task_create.c

```diff
@@ -91,7 +92,8 @@ static int nxthread_create(FAR const char *name, uint8_t ttype,
 
   /* Initialize the task */
 
-  ret = nxtask_init(tcb, name, priority, stack_ptr, stack_size, entry, argv);
+  ret = nxtask_init(tcb, name, priority, stack_ptr, stack_size, entry, argv,
+                    env_get());
```

Comment:
it's more simple to handle null pointer in the env_dup/env_set

Reply:
This is exactly how I did it in the first version of this PR so no problem

Reply:
Done